ui-manchette-with-spacetimechart/src/stories/base.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useRef } from 'react';
+
+import { Manchette } from '@osrd-project/ui-manchette';
+import {
+  type OperationalPointType,
+  type ProjectPathTrainResult,
+} from '@osrd-project/ui-manchette/dist/types';
+import { PathLayer, SpaceTimeChart } from '@osrd-project/ui-spacetimechart';
+import type { Meta } from '@storybook/react';
+
+import '@osrd-project/ui-core/dist/theme.css';
+import '@osrd-project/ui-manchette-with-spacetimechart/dist/theme.css';
+
+import { SAMPLE_PATH_PROPERTIES_DATA, SAMPLE_PATHS_DATA } from '../assets/sampleData';
+import useManchettesWithSpaceTimeChart from '../hooks/useManchetteWithSpaceTimeChart';
+
+const OperationalPointListData = SAMPLE_PATH_PROPERTIES_DATA.operational_points ?? [];
+const PathData = SAMPLE_PATHS_DATA ?? [];
+
+type ManchetteWithSpaceTimeWrapperProps = {
+  operationalPoints: OperationalPointType[];
+  projectPathTrainResult: ProjectPathTrainResult[];
+  selectedProjection: number;
+};
+const DEFAULT_HEIGHT = 561;
+
+const ManchetteWithSpaceTimeWrapper = ({
+  operationalPoints,
+  projectPathTrainResult,
+  selectedProjection,
+}: ManchetteWithSpaceTimeWrapperProps) => {
+  const manchette = useRef<HTMLDivElement>(null);
+  const { manchetteProps, spaceTimeChartProps } = useManchettesWithSpaceTimeChart(
+    operationalPoints,
+    projectPathTrainResult,
+    manchette,
+    selectedProjection
+  );
+
+  return (
+    <div className="manchette-space-time-chart-wrapper">
+      <div
+        className="header bg-ambientB-5 w-full border-b border-grey-30"
+        style={{ height: '40px' }}
+      ></div>
+      <div
+        ref={manchette}
```

Comment:
This is not the manchette ref, but the ref of the container of the spaceTimeChart and the manchette. Another possible name would be containerRef ?
```suggestion
        ref={spaceTimeChartWithManchetteRef}
```

Reply:
done but with the name of the project